lib/irb/default_commands.rb

```diff
@@ -236,6 +229,27 @@ def self.load_command(command)
       nil
     end
 
+    def self.install_helper_methods
+      HelperMethod.helper_methods.each do |name, helper_method_class|
+        define_method name do |*args, **opts, &block|
+          verbose, $VERBOSE = $VERBOSE, nil
```

Comment:
What kind of warning need to be suppressed here?
Shouldn't we show a warning occurred inside `helper.execute` implementation?

Reply:
I want to memoize the helper method instance, but then in Ruby 2.7 it'd cause the ivar not initialized warning.